dev-libs/mpvqt/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="project">
+        <email>kde@gentoo.org</email>
+        <name>Gentoo KDE Project</name>
+    </maintainer>
+    <upstream>
+        <remote-id type="kde-invent">libraries/ktextaddons</remote-id>
```

Comment:
that looks like a leftover

Reply:
Oops, looks like that collided with something else I had locally. Removed